src/Umbraco.Web.BackOffice/Controllers/LogViewerController.cs

```diff
@@ -135,6 +137,13 @@ public IEnumerable<SavedLogSearch> DeleteSavedSearch(SavedLogSearch item)
             return _logViewer.DeleteSavedSearch(item.Name, item.Query);
         }
 
+        [HttpGet]
+        public ReadOnlyDictionary<string, LogEventLevel> GetLogLevels()
+        {
+            return _logViewer.GetLogLevels();
+        }
+
+        [Obsolete("Please use GetLogLevels() instead")]
```

Comment:
Specify removal date here too

```suggestion
        [Obsolete("Please use GetLogLevels() instead. Scheduled for removal V11.")]
```